wiki/Beatmap_Editor/Beat_Snap_Divisor/de.md

```diff
@@ -25,14 +65,60 @@ Weiße Linie: Viervierteltakt
 
 Grüne Linie: Sie.
 
-| Halber Grundschlag | ![](1_1_m.jpg "1_1_m.jpg")                        | ![](1_2_m.jpg "1_2_m.jpg")                               | ![](1_3_m.jpg "1_3_m.jpg")                    | ![](1_4_m.jpg "1_4_m.jpg")                               |
-|--------------------|---------------------------------------------------|----------------------------------------------------------|-----------------------------------------------|----------------------------------------------------------|
-| Divisor            | 1/1                                               | 1/2                                                      | 1/3                                           | 1/4                                                      |
-| Beat               | Volle-Beats                                       | Halbe-Beats                                              | Tripel                                        | Viertel-Beats                                            |
-| Tick Noten         | Nur weiße Ticks                                   | Fügt rote Ticks hinzu                                    | Hat zwei lila Ticks zwischen jeden roten Tick | Fügt gelbe Ticks als Ergänzung zu roten und blauen hinzu |
-| Noten              | Am besten für Easy Schwierigkeitsstufen geeignet. |                                                          | Compound Timing                               | Nützlich für generelles Mapping                          |
-| Halber Grundschlag | ![](1_6_m.jpg "1_6_m.jpg")                        | ![](1_8_m.jpg "1_8_m.jpg")                               | ![](1_12_m.jpg "1_12_m.jpg")                  | ![](1_16_m.jpg "1_16_m.jpg")                             |
-| Divisor            | 1/6                                               | 1/8                                                      | 1/12                                          | 1/16                                                     |
-| Beat               | Doppelte-Tripel                                   | 8. Beats                                                 | 12. Beats                                     | 16. Beats                                                |
-| Tick Noten         | Hat zwei lila Ticks zwischen jeden roten Tick     | Fügt gelbe Ticks als Ergänzung zu roten und blauen hinzu |                                               |                                                          |
-| Noten              | Compound timing                                   |                                                          |                                               |                                                          |
+### Divisor 1/1
+
+![](/wiki/shared/1_1_m.jpg)
+
+- Volle-Beats
+- Nur weiße Ticks
+- Am besten für Easy Schwierigkeitsstufen geeignet.
+
+### Divisor 1/2
+
+![](/wiki/shared/1_2_m.jpg)
```

Comment:
L78: Add text on ``[]`` to phrase which supposed to be there if the image doesn't load